include/mbgl/style/sources/raster_dem_source.hpp

```diff
@@ -10,24 +10,10 @@ class AsyncRequest;
 
 namespace style {
 
-class RasterDEMSource : public Source {
+class RasterDEMSource : public RasterSource {
 public:
     RasterDEMSource(std::string id, variant<std::string, Tileset> urlOrTileset, uint16_t tileSize);
-    ~RasterDEMSource() final;
```

Comment:
Keep the destructor definition with `default` implementation

Reply:
Does it not inherit the default destructor from `RasterSource`?

Reply:
It does. It's not necessary.